lms/services/lti_grading/interface.py

```diff
@@ -1,8 +1,37 @@
 class LTIGradingService:  # pragma: no cover
-    """Service for sending grades back to the LMS."""
+    """
+    Service for sending grades back to the LMS.
 
-    def __init__(self, grading_url):
-        self.grading_url = grading_url
+    From https://www.imsglobal.org/spec/lti-ags/v2p0#nomenclature:
+
+    Line item
+        A line item is usually a column in the tool platform's gradebook; it is able to hold the results associated with a specific activity for a set of users.
+
+    Line item container
+        A line item container has an array of line items.
+
+    Result
+        A result is usually a cell in the tool platform's gradebook; it is unique for a specific line item and user.
+
+    Score
+        A score represents the last score obtained by the student for the tool's activity. It also exposes the current status of the activity (like completed or in progress), and status of the grade.
+
+
+    These are the LTI1.3 concepts but with the exception of line item container they have a counter part in LTI 1.1:
+
+    https://www.imsglobal.org/spec/lti-ags/v2p0#migrating-from-basic-outcomes-service
+    """
+
+    def __init__(self, line_item_url: str, line_item_container_url: str):
+        self.line_item_url = line_item_url
+        """
+        Identifies one line item to read/write grades to.
+
+        In LTI 1.1 this maps to the equivalent `lis_outcome_service_url` parameter.
+        """
+
+        self.line_item_container_url = line_item_container_url
+        """Identifies a container that might hold many line items."""
```

Comment:
```suggestion
        """
        Initialize the service.
        
        :param line_item_url: Identifies one line item to read/write grades 
            to. In LTI 1.1 this maps to the equivalent 
            `lis_outcome_service_url` parameter.
        :param line_item_container_url: Identifies a container that might hold 
            many line items
        """
        self.line_item_url = line_item_url
        self.line_item_container_url = line_item_container_url
```

This looks a bit weird? Not sure that it's wrong, but it looks like the way we document class level attributes, but in a method.

I think it would be good to pick one style or the other? If we added class level attributes we could document like this, or in a method I would expect to see `:param line_item_url:` etc.